Branch-SDK/src/main/java/io/branch/referral/NativeShareLinkManager.java

```diff
@@ -61,7 +64,9 @@ public void onLinkCreate(String url, BranchError error) {
             });
 
         } catch (Exception e) {
-            e.printStackTrace();
+            StringWriter errors = new StringWriter();
```

Comment:
I think this is the right approach, we should get rid of plain calls to e.printStackTrace() since that bypasses our logging infra. We can clean up other instances in another PR.